pkg/sql/vars.go

```diff
@@ -2025,6 +2025,23 @@ var varGen = map[string]sessionVar{
 		},
 		GlobalDefault: globalTrue,
 	},
+
+	// CockroachDB extension.
+	`pg_catalog_hide_hidden_columns`: {
```

Comment:
I am sad about the mixture between the term `Hidden` and `NOT VISIBLE` in the code base. Given the external API is `NOT VISIBLE`, my sense is this should be `pg_catalog_hide_not_visible_columns`. I'd support a follow-up task to change some names in some internal go interfaces.